SQL_module/SubjectForm_Task2/db_manipulation.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+    include "./db_connection.php";
+
+    $table_name = "electives";
+
+    function insertInTable($title, $description, $lecturer){
+        try {
+
+            $connection = db_connection();
+            $sql = "INSERT INTO electives(title,description,lecturer) VALUES (:title, :description, :lecturer)";
+    
+            $preparedSql = $connection->prepare($sql) or die("Failed to prepare sql query.");
+            $preparedSql->bindParam(':title', $title);
+            $preparedSql->bindParam(':description', $description);
+            $preparedSql->bindParam(':lecturer', $lecturer);
+            
+            $preparedSql->execute() or die("Filed to execute sql query.");   
+        }
+        catch(PDOException $error) {
+            echo $error->getMessage();
+        }
+    }
+
+    function printContentOfTable(){
+        
+        global $table_name;
+        try {
+            $connection = db_connection();
+            $sql = "SELECT * FROM $table_name";
```

Comment:
There is no problem currently with this approach in this situation, but it is not the best idea to pass raw values to sql script that is being executed later usually. Just be mindful when you do it, if you are doing it on purpose.